ci/test/00_setup_env_win64_ucrt.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2025-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export CONTAINER_NAME=ci_win64_ucrt
```

Comment:
nit: Could consider naming this `ci_win64` (and the file), as this will be the default eventually, going forward.

Reply:
Thanks! [Reworked](https://github.com/bitcoin/bitcoin/pull/33764#issuecomment-3482770283).